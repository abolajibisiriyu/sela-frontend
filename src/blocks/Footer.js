import styled from "styled-components";

const Footer = styled.div`
  width: 90%;
  margin: 0 auto;
  display: flex;
  align-items: center;
  justify-content: center;
  border-top: 1px solid rgba(0, 0, 0, 0.3);
  padding: 47px 0px;

  color: #000000;
  font-weight: 600;
  opacity: 0.3;
  font-size: 1.2rem;
`;

export default Footer;
