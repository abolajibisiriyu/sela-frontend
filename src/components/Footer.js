import React from "react";

function AppFooter(props) {
  return <div />;
}

export default AppFooter;
