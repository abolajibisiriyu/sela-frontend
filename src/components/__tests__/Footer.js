import React from "react";
import renderer from "react-test-renderer";

import Footer from "../Footer";

describe("Footer Component", () => {
  it("renders correctly", () => {
    const component = renderer.create(<Footer />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
