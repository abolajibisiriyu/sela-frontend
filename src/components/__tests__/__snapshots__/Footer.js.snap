// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders correctly 1`] = `
<div
  className="sc-bdVaJa kyiZwp"
>
  (c) Sela. 
  2019
</div>
`;
